tagController.js

```diff
@@ -116,16 +157,20 @@ function tagList(tagRowId, tagRowSelect){
     tagRowSelect.innerHTML = "";
 
     let optionItr;
-    for(optionItr = 0; optionItr < tagData.tags.length; optionItr++){
-        let tagItem = tagData.tags[optionItr];
+    for(let tagOptionIterator of tagData.tags){
+        let tagItem = tagOptionIterator;
```

Comment:
you can just write `let tagItem of tagData.tags` you don't need the extra variable